third_party/png.BUILD

```diff
@@ -35,6 +35,7 @@ cc_library(
     ],
     includes = ["."],
     linkopts = ["-lm"],
+    copts = ["-DPNG_ARM_NEON_OPT=0"],
```

Comment:
Do we need to conditionally add this on aarch64?
I feel like there are certain platforms, where we may want neon optimizations?

Reply:
this patch only need for aarch64. 
https://github.com/tensorflow/tensorflow/issues/18643
https://github.com/opencv/opencv/issues/7600
I think maybe it need to redefine.